test/test_container_httpd.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+import time
+
+import pytest
+
+from pathlib import Path
+
+from container_ci_suite.container_lib import ContainerTestLib
+from container_ci_suite.utils import check_variables, ContainerTestLibUtils
+
+if not check_variables():
+    print("At least one variable from OS, VERSION is missing.")
+    sys.exit(1)
+
+TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
+
+VERSION = os.getenv("VERSION")
+OS = os.getenv("OS")
+IMAGE_NAME = os.getenv("IMAGE_NAME")
+if not IMAGE_NAME:
+    print(f"Built container for version {VERSION} on OS {OS} does not exist.")
+    sys.exit(1)
+print(IMAGE_NAME)
+
+image_name = IMAGE_NAME.split(":")[0]
+image_tag = IMAGE_NAME.split(":")[1]
+test_dir = os.path.join(os.getcwd())
+print(f"Test dir is: {TEST_DIR}")
+
+
+@pytest.fixture(scope="module")
+def app(request):
+    app = ContainerTestLib(image_name=IMAGE_NAME, s2i_image=True)
+    # app_name = os.path.basename(request.param)
+    yield app
+    pass
```

Comment:
I'm not sure if `pass` has any meaning here - yield will pause the function execution and jump out of it; when it's resumed, it could go straight to the cleaning part. (maybe cc @frenzymadness?)

Reply:
Good point. It does not make sense. Like it is done here: https://github.com/sclorg/container-ci-suite/blob/main/container_ci_suite/utils.py#L583

Reply:
Fixed by https://github.com/sclorg/httpd-container/pull/234/commits/754ce20daacda1180b14e3b551c5bb87fc08454a